gobase/command/command.go

```diff
@@ -27,18 +28,22 @@ func Command(c *cli.Context) error {
 			break
 		case "request":
 			generate.Request(c)
+		default:
+			fmt.Println("Command Does Not Exist")
 		}
 
 	case "db":
 		postgres := instance.Postgres()
 		defer postgres.Close()
 		switch secondArgs {
 		case "fresh":
-			db.Fresh(postgres)
+			db.Fresh(postgres, schemaName)
```

Comment:
name is not consistent please check: 
https://github.com/wincentrtz/gobase/pull/3/files